CONTRIBUTING.rst

```diff
@@ -0,0 +1,149 @@
+Team
+====
+
+* Team name must not contain odoo or openerp
+* Team name for localization is "Belgium Maintainers" for Belgium
+
+Project / Repository
+====================
+
+* Project name must not contain odoo or openerp
+* Project name for localization is "l10n_belgium" for Belgium
+* Project name for connectors is "connector-magento" for Magento connector
+
+Branch
+======
+
+
+Module
+======
+
+* Prefer the use of the singular form in module name (or use 'multi')
+* Use the description template (rst or html) provided on http://github.com/oca/maintainer-tools
+
+Python Code
+===========
+
+* Follow Python PEP 8: https://www.python.org/dev/peps/pep-0008/ 
+* Do not use deprecated features
+
+XML Code
+========
+
+JS Code
+=======
+
+Menu
+====
+
+Reporting
+=========
+
+Translation
+===========
+
+Security
+========
+
+Tests
+=====
+
+Demo data
+=========
+
+Migration
+=========
+
+Issue
+=====
+
+* Issues are used for blueprints and bugs.
+
+Commit
+======
+
+* Write a helpful commit message
+* Use a commit tag in each message. This tag should be one of:
```

Comment:
I don't think we should ask for these "commit tags" in the conventions. It is not anymore widely used since we moved on github. Personally I always preferred to have this kind of information well formed in the sentence itself ("Fixes this", "Refactor that"). When I was using those tags, I observed that it was often difficult to choose one or another (especially between IMP, FIX, REF).